src/app/shared/interfaces.routes.ts

```diff
@@ -0,0 +1,9 @@
+import { Request, Response } from "express";
```

Comment:
You can use `export * from './file';` to export directly all the variables from a file.